a10_octavia/controller/worker/tasks/virtual_port_tasks.py

```diff
@@ -41,7 +41,8 @@ def set(self, set_method, loadbalancer, listener, ssl_template=None):
 
         virtual_port_templates = {}
         template_virtual_port = CONF.listener.template_virtual_port
-        virtual_port_templates['template-virtual-port'] = template_virtual_port
+        virtual_port_templates['template-virtual-port'] = utils.get_template_name(
```

Comment:
There isn't a need to go so far as a unique utility function. We are already defining a variable here so we might as well use it for real.

For line length concerns we can cut down `virtual_port_templates` to `vport_templates`

```python
template_vport = CONF.listener.template_virtual_port
vport_templates['template-virtual-port'] = template_vport if template_vport != "None" else None
```

Later we can just use name the template vars better for example

```python
template_http = CONF.listener.template_http
vport_templates['template-http'] = template_http if template_http != "None" else None
```

Reply:
Not used ternary operator, added seperate condition.